layouts/events/single.html

```diff
@@ -0,0 +1,154 @@
+    style="background: radial-gradient(#063053e5, #062e52e5);">
+    <h1 class="inter-700 text-48 text-white text-center z-4">{{ .Title | markdownify }}</h1>
+    <div class="position-absolute ellipses">
+        <img src="/images/essentials/ellipses.png" alt="" class="ellipseW">
+    </div>
+</section>
+{{/* End Hero Section */}}
+
+{{/* Main Section */}}
+<section class="section bg-dark-100">
+    <div class="container-lg">
+        <div class="row justify-content-between">
+            <div class="col-lg-8">
+                <div class="bg-white p-4 rounded-3 shadow-sm">
+                    {{ if .Params.image }}
+                    <div class="mb-4">
+                        <img src="{{ .Params.image }}" alt="{{ .Title }}" class="img-fluid rounded-3 w-100">
+                    </div>
+                    {{ end }}
+                    <div class="d-flex flex-wrap gap-4 mb-4">
+                        {{ with .Date }}
+                        <div class="d-flex gap-2 align-items-center">
+                            <div class="icons">
+                                <i class="fas fa-calendar-alt text-primary" style="font-size: 1.2rem;"></i>
+                            </div>
+                            <p class="inter-400 text-16 dark-600 mb-0 pt-1">
+                                {{ . | time.Format "January 02, 2006" }}
+                                {{ if $.Params.endDate }}
+                                  {{ $startDate := . | time.Format "2006-01-02" }}
+                                  {{ $endDate := time.AsTime $.Params.endDate | time.Format "2006-01-02" }}
+                                  {{ if ne $startDate $endDate }}
+                                  - {{ time.AsTime $.Params.endDate | time.Format "January 2, 2006" }}
+                                  {{ end }}
+                                {{ end }}
+                            </p>
+                        </div>
+                        {{ end }}
+                        {{ with .Params.location }}
+                        <div class="d-flex gap-2 align-items-center">
+                            <div class="icons">
+                                <i class="fas fa-map-marker-alt text-primary" style="font-size: 1.2rem;"></i>
+                            </div>
+                            <p class="inter-400 text-16 dark-600 mb-0 pt-1">{{ . }}</p>
+                        </div>
+                        {{ end }}
+                        {{ with .Params.eventType }}
+                        <div class="d-flex gap-2 align-items-center">
+                            <div class="icons">
+                                <i class="fas fa-tag text-primary" style="font-size: 1.2rem;"></i>
+                            </div>
+                            <p class="inter-400 text-16 dark-600 mb-0 pt-1">{{ . }}</p>
+                        </div>
+                        {{ end }}
+                    </div>
```

Comment:
I like the info on the right! Thanks. Now I feel like this is kind of redundant and could be removed. Up to you though.

```suggestion
```